src/vehicle.cpp

```diff
@@ -1340,28 +1297,30 @@ const vpart_info& vehicle::part_info (int index, bool include_removed) const
 
 // engines & alternators all have power.
 // engines provide, whilst alternators consume.
-int vehicle::part_power(int const index, bool const at_full_hp) const
+int vehicle::part_power(int const index, bool at_full_hp) const
 {
-    if( !part_flag(index, VPFLAG_ENGINE) &&
-        !part_flag(index, VPFLAG_ALTERNATOR) ) {
-       return 0; // not an engine.
-    }
+    const vehicle_part &vp = parts[index];
+    int pwr = 0;
 
-    const vehicle_part& vp = parts[ index ];
+    ///\EFFECT_STR increases power output of manual engines
+    if( part_flag( index, "MUSCLE_LEGS" ) ) {
+        // average man (STR 10) = 1/3 hp
+        pwr = g->u.str_cur * 0.033 * 745.7;
```

Comment:
Would be much better to push those to json.
Real life constants in the code are hard to adjusts and are a big code smell. Same real life constants in the jsons are way more manageable.